<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Total Finder</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="TotalFinder::MyTabWidget" name="LeftTabs">
         <property name="styleSheet">
          <string notr="true">QTabWidget::pane {
   position: absolute;
}

QTabWidget::tab-bar {
    alignment: center;
    min-width: 15000px;
    border: 1px solid red;
}

QTabBar::tab {
    min-width: 8ex;
    padding: 2px;
    text-align: center;
    color: rgb(63, 63, 80);
    border-right: 1px solid rgb(154, 154, 154);
    background-color: rgb(184, 184, 183);
}

QTabBar::tab:selected {
  background-color: rgb(204, 204, 203);
}
</string>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TotalFinder::MyTabWidget" name="RightTabs">
         <property name="styleSheet">
          <string notr="true">QTabWidget::pane {
   position: absolute;
}

QTabWidget::tab-bar {
    alignment: center;
    min-width: 15000px;
    border: 1px solid red;
}

QTabBar::tab {
    min-width: 8ex;
    padding: 2px;
    text-align: center;
    color: rgb(63, 63, 80);
    border-right: 1px solid rgb(154, 154, 154);
    background-color: rgb(184, 184, 183);
}

QTabBar::tab:selected {
  background-color: rgb(204, 204, 203);
}
</string>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTF">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="ActionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ActionHelp"/>
    <addaction name="ActionAbout"/>
   </widget>
   <addaction name="menuTF"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionTest_item">
   <property name="text">
    <string>test item</string>
   </property>
  </action>
  <action name="ActionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="ActionHelp">
   <property name="text">
    <string>Keymap</string>
   </property>
  </action>
  <action name="ActionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TotalFinder::MyTabWidget</class>
   <extends>QTabWidget</extends>
   <header>tab_manager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
